recipes/mitos/mitos1/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "mitos" %}
```

Comment:
Do we need to archive this?
Basically, there are two reasons - a) we have things depending on `mitos <2` or b) there will be future releases of `mitos 1`. If those don't apply - just remove this.

Reply:
I will provide bugfix releases for MITOS1 if necessary.

Reply:
Think that that's likely? You can always resurrect the old recipe from git. I'm just trying to keep the number of sub-recipes down. Most of us don't have time to support old branches, so it's unusual for backports to actually happen.

The real reason for the sub-recipes is that we have some tools that depend on older versions. So if something pins `blast=2.7.*` then we need to keep the blast 2.7 recipe around in case we need to update e.g. openssl and rebuild.